### ISSUE TITLE
Signal notifications don't come to watch

### ISSUE THREAD

**Issue by skorokithakis** [AmazMod/AmazMod#334]
Notifications from the Signal app won't get pushed to the watch. Notifications from other apps work fine, but Signal doesn't. The notification log says "GROUP" next to Signal's notifications.

**Comment by lfom** [comment]
Is "ungroup notifications" enabled in AmazMod settings?

**Comment by lfom** [comment]
If it is listed as "grouped" in the log, they will be sent to watch only and if only "ungrouped" is enabled.

**Comment by GreatApo** [comment]
Like lfom said, because of the type of the notification, it needs to be ungrouped to be send again.

If the problem persist re-open the issue.

**Comment by alfaflo** [comment]
Hello,

I would like to re-open this topic as I'm facing the same problem.
I've enabled "ungroup notifications" in AmazMod settings.
I'm using latest AmazMod apk 1.1.3 (play store beta channel) and  AmazMod Service 1880 on watch.
Other "group" notifications work well but notifications from "Signal" do not appear on watch.

Thank you.

**Comment by diotto** [comment]
@alfaflo did you enabled local notifications in amazmod preferences?

**Comment by alfaflo** [comment]
> @alfaflo did you enable local notifications in amazmod preferences?

Hello Diotto,
Yes, "local notifications" are enabled.

**Comment by diotto** [comment]
Please, can you send us a screen of Stats > Notifications Logs just after some signal notification arrives? I would like to understand what is filtering it

**Comment by alfaflo** [comment]
![screenshot_20190213_133757_com edotassi amazmod](https://user-images.githubusercontent.com/47593430/52714310-d4e83400-2f99-11e9-855f-b81e33ef2d90.jpg)

**Comment by alfaflo** [comment]
"org.thoughtcrime.securesms" -> App "Signal"

**Comment by diotto** [comment]
I don't understand German but it looks like you have "Disable Notifications when screen is on" enabled and signals messages arrived with screen on. Can you show a log with screen off or disable this option in Amazmod preferences?

**Comment by alfaflo** [comment]
> I don't understand German but it looks like you have "Disable Notifications when screen is on" enabled and signals messages arrived with screen on. Can you show a log with screen off or disable this option in Amazmod preferences?

Yes, this option is enabled. That's the reason why a made a big screenshot showing several events. So for the latest on top (pls check date of event) there is no "screen on" ("BIDLSCHIRM EIN") note.
If you need more information, pls let me know.

**Comment by diotto** [comment]
@alfaflo the notifications on top are showing GROUP (if i understood). Is the preference "Enable Notifictations Ungroup" checked in Amazmod Preferences ?

**Comment by alfaflo** [comment]
Yes, this option is enabled. Other apps with "grouped" notifications (e.g. my calendar or "one football" work and if I switch off "ingroup" those notifications are not synced)

**Comment by 1immortal** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Features](https://trello.com/c/p6NGIJCP/2-features)

**Comment by 1immortal** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Signal notifications don't come to watch](https://trello.com/c/MrfNCruH/16-signal-notifications-dont-come-to-watch)
